attributes/default.rb

```diff
@@ -56,6 +57,9 @@
 default['cfncluster']['slurm']['version'] = '20.02.1'
 default['cfncluster']['slurm']['url'] = 'https://download.schedmd.com/slurm/slurm-20.02.1.tar.bz2'
 default['cfncluster']['slurm']['sha1'] = '91704d8c73bf1a6eab64ccde3f3d03b85188672a'
+# ParallelCluster slurm variables
+default['cfncluster']['slurm']['queue_config_s3_uri'] = nil
+default['cfncluster']['slurm']['queue_config_path'] = "#{node['cfncluster']['configs_dir']}/slurm_config.json"
```

Comment:
nit: in the end this is not a slurm_config. Shall we find a more generic name for it? `queues_config` or something else that does not have slurm in it?